src/tips/fixtures/tiptypes.json

```diff
@@ -154,7 +154,7 @@
     "pk": 16,
     "fields": {
       "name": "tour",
-      "priority": 1,
+      "priority": 0,
```

Comment:
Very nice.  And, it makes PR #337 to shuffle the priorities around unnecessary*.

\<curmudgeon>And here I thought that zero was neither positive nor negative.  Django's `PositiveSmallIntegerField` says otherwise\</curmudgeon>

\* for now...

Reply:
One of those areas where math ≠ computer science, I guess?